requirements.txt

```diff
@@ -1,4 +1,5 @@
 jupyter
+ipykernel<5.*
```

Comment:
I'm assuming testing only on Python 2.7 for now?

Reply:
Its working for me locally using `ipykernel==4.10.0` and python 3.5.

Its based on: https://github.com/jupyter/jupyter/issues/370

Reply:
is this still needed (now that the `jupyter` metapackage has been replaced by `notebook` in `requirements.txt`)?
what was the main reason behind it?